### ISSUE TITLE
Freeze Encrypting error

### ISSUE THREAD

**Issue by calex80** [optiv/Freeze.rs#4]
Hi, I am looking to find a way to use freeze to encrypt cobalt payloads, I was attempting with golang version but didn't work, I now, I am trying to make this Rust version work.

I am facing this error posted below, it says `"No such file or directory" }', build/src/build.rs:23:10` 
Could you help to understand what I am missing?
Thank you

![image](https://user-images.githubusercontent.com/58860205/236472122-9bac9290-490e-4c2a-97c7-fe503466cd03.png)

**Comment by Tylous** [comment]
Hello, you are not using raw shellcode, you are using a pre-compiled binary.

**Comment by calex80** [comment]
Yes, my bad, you are right, now it's working perfectly!!! I mean the compile process

**Comment by q99266** [comment]
it says
“thread 'main' panicked at 'Failed to open Cargo.toml: Os { code: 2, kind: NotFound, message: "No such file or directory" }', build/src/build.rs:30:10
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace”

Could you help to understand what I am missing?
Thank you
